storage/tests/unit/test_blob.py

```diff
@@ -55,7 +55,8 @@ def test_ctor_with_encoded_unicode(self):
         blob_name = b'wet \xe2\x9b\xb5'
         blob = self._make_one(blob_name, bucket=None)
         unicode_name = u'wet \N{sailboat}'
-        self.assertNotEqual(blob.name, blob_name)
+        self.assertNotIsInstance(blob.name, bytes)
```

Comment:
I was under the impression that "text" means `str` (i.e. what you get when `a = 'b'`), but `str == bytes` in Py2 so this test wouldn't succeed there.